WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -424,6 +452,18 @@ - (void)showViewAdminForBlog:(Blog *)blog
     [[UIApplication sharedApplication] openURL:[NSURL URLWithString:dashboardUrl]];
 }
 
+- (void)showRemoveSiteForBlog:(Blog *)blog
+{
+    NSString *model = [[UIDevice currentDevice] localizedModel];
+    NSString *title = [NSString stringWithFormat:NSLocalizedString(@"Are you sure you want to continue?\n All site data will be removed from your %@.", @"Title for the remove site confirmation alert"), model];
```

Comment:
Maybe, but then are we saying we're not as worried about the the message looking weird in other localizations?  Maybe there's not a good way to handle it for all cases. :shrug: